src/envoy/http/backend_routing/filter.h

```diff
@@ -37,6 +37,16 @@ class Filter : public Envoy::Http::PassThroughDecoderFilter,
                                                  bool) override;
 
  private:
+  std::string translateConstPath(
```

Comment:
These two can be anonymous function

Reply:
You mean a lambda? I don't see the benefit of that.

Reply:
Sorry, function in anonymous space.

Reply:
Oh, `translateConstPath` uses the filter state member variable. So it must be a member function. We should keep both here for consistency.